src/solids4FoamModels/numerics/vfvCellPoint/vfvcCellPoint.C

```diff
@@ -894,7 +894,7 @@ tmp<volScalarField> interpolate
         const labelList& curCellPoints = cellPoints[cellI];
 
         // Number of points in current cell
-        const scalar nPoints = curCellPoints.size();
+        const scalar& nPoints = curCellPoints.size();
```

Comment:
@federico081997: `nPoints` here should be an `int` rather than a scalar.

In addition, generally, there is no performance gain in taking a reference for quantities as small (in memory footprint) as a scalar/int/bool/etc. Of course, there are cases where you may want to take a reference to edit the quantity (not here, since it is `const`).